src/components/Header/Logo/index.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+import image from "./img/logo1.png";
+
+function Logo() {
+  return (
+    <LogoWraper>
+      <a href="">
```

Comment:
Better not to have empty `href`'s

```suggestion
      <a href="#">
```